library/src/skins/default/src/sass/tool.scss

```diff
@@ -248,3 +248,7 @@ iframe[type="application/pdf"] {
 .popover-body {
     color: var(--sakai-text-color-1);
 }
+
+.si-x-square-fill::before {
+    color: red;
```

Comment:
Won't this mean that anytime someone uses this icon, it will be red? Should it not be called close icon, or something?